src/test/java/org/minions/devfund/alvaro/WhackAMoleTest.java

```diff
@@ -0,0 +1,97 @@
+package org.minions.devfund.alvaro;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * test for whack a mole game.
+ */
+public class WhackAMoleTest {
+    private WhackAMole whackAMole;
+
+    /**
+     * constructor .
+     */
+    public WhackAMoleTest() {
+        final int gridDimension = 3;
+        whackAMole = new WhackAMole(2, gridDimension);
+    }
+
+    /**
+     * print whacks and moles.
+     */
+    @Test
+    public void printGridTest() {
+        whackAMole.place(0, 0);
+        whackAMole.place(1, 1);
+        whackAMole.place(2, 2);
+        whackAMole.whack(0, 0);
+        whackAMole.whack(1, 1);
+        whackAMole.printGrid();
```

Comment:
Since your pull request is still open 
please write your unit tests something like this.
`
Running: testConstructorGrid
Finished: testConstructorGrid
Running: testConstructorNumOfAttempts
Finished: testConstructorNumOfAttempts
Running: testPlaceFail
Finished: testPlaceFail
Running: testWhackSucceedAttempts
Finished: testWhackSucceedAttempts
Running: testWhackFailAttempts
Finished: testWhackFailAttempts
Running: testPlaceSucceed
Finished: testPlaceSucceed
Running: testWhackFailMolesLeft
Finished: testWhackFailMolesLeft
Running: testWhackSucceedMolesLeft
Finished: testWhackSucceedMolesLeft
Tests run: 8, Passed: 8, Failed: 0
`